modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView/VersionsContent.tsx

```diff
@@ -113,12 +116,21 @@ const VersionsContent = ({
 					))}
 				</ul>
 			)}
+			{viewVersionsURL && (
+				<ClayLink
+					className="d-flex justify-content-center mt-3"
+					href={viewVersionsURL}
+				>
+					{Liferay.Language.get('view-more')}
```

Comment:
If we have less than the default (10?) the copy `see more` seems not so accurate, as the user will see the same in the linked page. Should we hide the button in that case?

Reply:
Yeah, I put "See more" temporarily but it is something that we should talk with Deborah

Reply:
cc. @deborah-silva